lib/common/asar.js

```diff
@@ -275,21 +275,21 @@
     }
 
     const { statSync } = fs
-    fs.statSync = pathArgument => {
+    fs.statSync = (pathArgument, ...args) => {
       const { isAsar } = splitPath(pathArgument)
-      if (!isAsar) return statSync(pathArgument)
+      if (!isAsar) return statSync(pathArgument, ...args)
 
       // Do not distinguish links for now.
-      return fs.lstatSync(pathArgument)
+      return fs.lstatSync(pathArgument, ...args)
     }
 
     const { stat } = fs
-    fs.stat = (pathArgument, callback) => {
+    fs.stat = (pathArgument, ...args) => {
       const { isAsar } = splitPath(pathArgument)
-      if (!isAsar) return stat(pathArgument, callback)
+      if (!isAsar) return stat(pathArgument, ...args)
 
       // Do not distinguish links for now.
-      process.nextTick(() => fs.lstat(pathArgument, callback))
+      process.nextTick(() => fs.lstat(pathArgument, ...args))
```

Comment:
This won't work as `lstat` is hardcoded for two args just above here.

Refs: https://github.com/electron/electron/pull/15323/files#diff-4ca10576721b01fdda42127212bb00d6R255

Reply:
Curses